packages/patternfly-react/src/components/SessionTimeout/SessionTimeout.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon, MessageDialog } from '../../index';
+
+const SessionTimeout = props => (
+  <MessageDialog
+    show={props.timeLeft > 0 && props.timeLeft <= props.displayBefore}
+    primaryActionButtonContent={props.continueText}
+    primaryAction={props.continueFnc}
+    secondaryActionButtonContent={props.logoutText}
+    secondaryAction={props.logoutFnc}
+    primaryContent={props.primaryContent}
+    secondaryContent={props.secondaryContent}
+    onHide={props.logoutFnc}
+    icon={<Icon type="pf" name="warning-triangle-o" />}
+  />
+);
+
+SessionTimeout.propTypes = {
+  timeLeft: PropTypes.number.isRequired,
+  displayBefore: PropTypes.number,
+  continueText: PropTypes.string,
```

Comment:
We should keep this as MessageDialog and use PropTypes.node to be more flexible

Reply:
no problem. is that OK if I chage it to `continueContent`/`logoutContent` instead of `continueText`/`logoutText`?